src/Workspaces/VisualBasic/Portable/Rename/VisualBasicRenameRewriterLanguageService.vb

```diff
@@ -275,6 +277,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Rename
                             suffix = "Event"
                         End If
 
+                        If fieldSymbol.AssociatedSymbol IsNot Nothing AndAlso
+                           fieldSymbol.AssociatedSymbol.IsKind(SymbolKind.Property) AndAlso
+                           fieldSymbol.Name = "_" + fieldSymbol.AssociatedSymbol.Name Then
```

Comment:
Not currently, but the docs give a warning that AssociatedSymbol may come to mean more things as the language revs. These checks don't hurt, and they might protect against future meanings of AssociatedSymbol.